otwcode/otwarchive
# AO3-6048 Add linters to main Gemfile (and install pwgen)

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests. (@redsummernight said it was ok, I swear!)

## Issue

https://otwarchive.atlassian.net/browse/AO3-6048

## Purpose

1. Adds pwgen to the installed packages of the docker images (for easier creation of admin users for dev/testing)
2. Incorporates the Gemfile.linters contents into the main Gemfile and removes the linters file

## Testing Instructions

These changes affect development only.

1. Build a new docker (web) image and verify that the [admin creation script](https://github.com/otwcode/otwarchive/wiki/Development-Data#creating-admins) can be used out-of-the-box without installing additional packages
2. Build a new docker (web) image and verify that the linting commands can be run out-of-the-box without installing additional packages

## References

N/A

## Credit

tlee911 (he/him)
